spec/datadog/ci/ext/environment_spec.rb

```diff
@@ -4,19 +4,24 @@
 require 'datadog/ci/ext/environment'
 
 RSpec.describe Datadog::CI::Ext::Environment do
-  describe '::tags' do
-    def self.match(env, tags)
-      it "matches tags from #{env}" do
-        ClimateControl.modify('HOME' => env['HOME']) do
-          expect(described_class.tags(env)).to eq(tags)
-        end
-      end
-    end
+  describe '.tags' do
+    subject(:tags) { described_class.tags(env) }
+    let(:env) { {} }
```

Comment:
Minor: This is not needed (since the inner context always defines it).

Reply:
👍 
This is also because I broke this feature out of a larger changeset.
This `let` will be reintroduced later, but won't conflict too much, so I'll remove it from this PR.